README.md

```diff
@@ -176,3 +176,8 @@ Development instructions for the CircleCI CLI can be found in [HACKING.md](HACKI
 ## More
 
 Please see the [documentation](https://circleci-public.github.io/circleci-cli) or `circleci help` for more.
+
+
+## Version Compatibility 
+
+As of homebrew version `0.1.24705` - we no longer support Server 3.x instances. In order to upgrade the CLI to the latest version, you'll need to update your instance of server to 4.x.
```

Comment:
Users don't have to use homebrew to install the CLI. Including the package manager could be confusing to end users.

```suggestion
As of version `0.1.24705` - we no longer support Server 3.x instances. In order to upgrade the CLI to the latest version, you'll need to update your instance of server to 4.x.
```

Is `0.1.24705` the last supported version of server 3.x instances? Or do we need to update this to include a usable version?

Reply:
`0.1.23845` will be the last support version for server 3.x. 

I'll update this to include this information.